test-packages/e2e-tests/package.json

```diff
@@ -7,14 +7,22 @@
   "private": true,
   "repository": "github:SAP/cloud-sdk-js",
   "scripts": {
-    "pretest:e2e": "yarn deploy && yarn start",
-    "posttest:e2e": "yarn stop",
-    "start": "pm2 start server.js || pm2 restart server.js",
-    "stop": "pm2 stop server.js",
+    "pretest:e2e": "yarn deploy && yarn start:service-server && yarn start:proxy-server && yarn start:mail-server",
+    "posttest:e2e": "yarn stop:service-server && yarn stop:service-server && yarn stop:mail-server",
+    "start:service-server": "pm2 start server.js || pm2 restart server.js",
```

Comment:
I would prefer if the start and stop functionality of the server would be handled in the `beforeAll` and `afterAll` phase of the test.

Reply:
I do not understand what you mean with `does not job`

Reply:
Sorry for the typo, I mean "Please check the line 10 and 11, that does the job.", that is executed "before" and "after" the tests.